tools/bowtie2.wdl

```diff
@@ -0,0 +1,355 @@
+        end_to_end: "If true, entire read must align; no clipping. Else, local alignment; ends might be soft clipped"
+        match_bonus: "bonus for match (0 for end-to-end or 2 for local)"
+        mismatch_penalty: "max penalty for mismatch; lower qual = lower penalty"
+        non_actg_penalty: "penalty for non-A/C/G/Ts in read/ref"
+        read_gap_open_extend: "read gap open, extend penalties"
+        ref_gap_open_extend: "ref gap open, extend penalties"
+        max_aln_report: "report up to N alns per read; MAPQ not meaningful"
+        report_all_alignments: "report all alignments; very slow, MAPQ not meaningful"
+        min_fragment_len: "minimum fragment length"
+        max_fragment_len: "maximum fragment length"
+        no_mixed: "suppress unpaired alignments for paired reads"
+        no_discordant: "suppress discordant alignments for paired reads"
+        dovetail: "concordant when mates extend past each other"
+        no_contain: "not concordant when one mate alignment contains other"
+        no_overlap: "not concordant when mates overlap at all"
+        time: "print wall-clock time taken by search phases"
+        write_unpaired_unaligned: "write unpaired reads that didn't align"
+        write_unpaired_aligned: "write unpaired reads that aligned at least once"
+        write_paired_discordant: "write pairs that didn't align concordantly"
+        write_paired_concordant: "write pairs that aligned concordantly at least once"
+        quiet: "print nothing to stderr except serious errors"
+        metrics_file: "write metrics to file"
+        metrics_stderr: "write metrics to stderr"
+        metrics_interval: "report internal counters & metrics every N seconds"
+        no_unal: "suppress SAM records for unaligned reads"
+        no_head: "suppress header lines, i.e. lines starting with @"
+        no_sq: "suppress @SQ header lines"
+        rg: "Read group record for SAM/BAM header"
+        addl_rg_text: "add <text> ('label:value') to @RG line of SAM header"
+        omit_sec_seq: "put '*' in SEQ and QUAL fields for secondary alignments."
+        sam_no_quane_trunc: "Suppress standard behavior of truncating readname at first whitespace at the expense of generating non-standard SAM."
+        xeq: "Use '='/'X', instead of 'M,' to specify matches/mismatches in SAM record"
+        soft_clipped_unmapped_tlen: "Exclude soft-clipped bases when reporting TLEN."
+        sam_append_comment: "Append FASTA/FASTQ comment to SAM record."
+        sam_opt_config: "Use <config>, example '-MD,YP,-AS', to toggle SAM optional fields."
+        threads: "Number of alignment threads to use"
+        reorder: "force SAM output order to match order of input reads"
+        memory_map: "use memory-mapped I/O for index; many 'bowtie's can share"
+        qc_filter: "filter out reads that are bad according to QSEQ filter"
+        seed: "seed for random number generator"
```

Comment:
I didn't realize until reading the manual, but this option only works with `--qseq` which I didn't (and don't think we should) implement.